pkg/cli/root.go

```diff
@@ -219,7 +219,7 @@ func validateConfiguration(config *config.Config) error {
 	if config.HTTP.RawAccessControl != nil {
 		if config.HTTP.Auth == nil || (config.HTTP.Auth.HTPasswd.Path == "" && config.HTTP.Auth.LDAP == nil) {
 			// checking for default policy only authorization config: no users, no policies but default policy
-			checkForDefaultPolicyConfig(config)
+			checkForAnonymousPolicyConfig(config)
```

Comment:
I find this function name confusing, it's not clear what `check` means.
Why not call it `validateAuthzPolicies` and let the comments/log messages inside give more details? And return the error in case there is a a bad configuration.